shared/util/kbfs.js

```diff
@@ -24,11 +24,3 @@ export function getTLF (isPublic: boolean, basedir: string): string {
     return `/private/${basedir}`
   }
 }
-
-export function cleanup (folderName: string): string {
-  if (!folderName) {
-    return ''
-  }
-
-  return folderName.replace(/\s/g, '').replace(/\.\./g, '').replace(/\//g, '').replace(/\\/g, '')
```

Comment:
I now use `path.resolve` to remove any .. and then check that it has the kbfs prefix.